package.json

```diff
@@ -5,17 +5,18 @@
   "browser": "dist/index.js",
   "module": "esm/index.js",
   "dependencies": {
-    "@babel/plugin-proposal-optional-chaining": "^7.14.5",
-    "@babel/runtime": "^7.18.0",
+    "@babel/plugin-proposal-optional-chaining": "^7.18.9",
+    "@babel/runtime": "^7.18.9",
+    "@patternfly/react-charts": "6.14.17",
```

Comment:
This is new 🤔 has one patternfly package been split up into two?

Reply:
Oh, sorry, I meant to make a comment about that. For now we need to lock the charts package, or it raises errors.
See also: https://github.com/RedHatInsights/insights-advisor-frontend/pull/981/files#r891133348